pkg/ccl/changefeedccl/retry.go

```diff
@@ -45,7 +47,7 @@ func testingUseFastRetry() func() {
 
 // reset retry state after changefeed ran for that much time
 // without errors.
-const resetRetryAfter = 10 * time.Minute
```

Comment:
Should we maybe keep this at a larger value (maybe some multiple of `maxBackoff`)? If the changefeed was only able to run for just over a minute, I'm not sure we'd want to restart the exponential backoff back at 500 ms.